### PR TITLE
LPS-154679 | LPS-155353 | LPS-155355

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-154679
https://issues.liferay.com/browse/LPS-155353
https://issues.liferay.com/browse/LPS-155355

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Review by rodrigopaulino — CHANGES_REQUESTED** [review]
Nice, @carolmariaabb! 👏🏻 
Just two minor requests below. Thanks!

**Comment by carolmariaabb** [comment]
ci:stop

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java [hunk 1]
```diff
@@ -290,7 +290,7 @@ public void testGetDataWithNumberOutput() throws Exception {
 			ddmDataProviderResponse.getOutputOptional(
 				outputParameterId, Number.class);
 
-		Assert.assertEquals(246, outputOptional.get());
+		Assert.assertEquals(247, outputOptional.get());
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/data/provider/test/DDMRESTDataProviderTest.java]: Done! Amazing idea, thanks for suggesting!

File: modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/jaxrs/exception/mapper/BaseDataEngineExceptionMapper.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
+import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
+
+import javax.ws.rs.core.Response;
+
+/**
+ * @author Carolina Barbosa
+ */
+public abstract class BaseDataEngineExceptionMapper<T extends Throwable>
```
[the remark this answers is not in the document]
Comment (carolmariaabb) [reply modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/jaxrs/exception/mapper/BaseDataEngineExceptionMapper.java]: Done!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Review by rodrigopaulino — APPROVED** [review]
Awesome! LGTM! 🎉

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:reevaluate:1267401_3974

**Pull request closed without merging**